### ISSUE TITLE
Non-embedded book covers not showing up

### ISSUE THREAD

**Issue by 0x4c47** [PaulWoitaschek/Voice#1665]
### Overview

When adding any of my audiobooks in Voice, no book covers saved as cover.jpg are showing up for them. I expect my book covers to show up.

My folder structure: `<audiobook root folder>/<author>/<book title>/<mp3 files and cover.jpg>`
cover.jpg is just a simple .jpg file and I tried a lot of different ones now. None of them shows up. What does show up though are embeeded cover images that are part of the mp3 file's ID3 tags. But for a book without embedded covers Voice's grey default cover image is showing.

My audiobooks are saved on my external SD card if that's relevant. I don't have any cover art issues with any music players and my music files on my external SD card though. (EDIT: Doesn't seem
to be relevant, see first comment.)

### How to reproduce

1. Add cover.jpg to your audiobook folder and remove embedded covers
2. Add audiobook root folder with structure described above
3. Add to library in Voice
4. See that no covers are showing up in the overview page except for the embedded covers

### Version

6.5.0

### Android Version

13

### Device Model

Pocophone F1

**Comment by 0x4c47** [comment]
For troubleshooting purposes I've moved the audiobooks to internal storage. Didn't make a difference, still no covers showing in Voice.

**Comment by xcy7e** [comment]
To my experience, once an audiobook is loaded, nothing will change afterwards.

Try renaming the folder so the book has to be loaded again.

Also verify the jpg files have correct mimeTypes and are not just renamed png's for example.

**Comment by 0x4c47** [comment]
Ok, I found the issue while trying what you've suggested: My directory structure.

I didn't have multiple audiobooks by the same author. That's why I didn't notice any problems with my folder structure `<audiobook root folder>/<author>/<book title>.

But Voice actually detected each author directory as a single audiobook (instead of the book title subdirectory) and only the ID3 tags lead to the correct view of the metadata in the interface (author and book name was correct there). Without the ID3 tags it would've probably shown the author's name as the book title. And with multiple audiobooks by the same author I probably would've noticed this earlier.

I changed my directory structure which is now `<audiobook root folder>/<author> - <book title>`.

Covers are now showing up correctly so I'm closing this issue.
